code/modules/admin/topic.dm

```diff
@@ -4991,6 +4991,14 @@ access_sec_doors,access_salvage_captain,access_cent_ert,access_syndicate,access_
 			if("can rotate")
 				new_value = input(usr,"0 - rotation disabled, 1 - rotation enabled","Shuttle editing",S.can_rotate) as num
 				S.can_rotate = new_value
+			if("destroy areas")
+				new_value = input(usr,"Allow this shuttle to crush into areas? Currently set to: [S.destroy_everything ? "True" : "False"]","Shuttle editing") as null|anything in list("CRUSH","No crush")
+				if(new_value == "CRUSH")
+					S.destroy_everything = TRUE
+				else if(new_value == "No crush")
+					S.destroy_everything = FALSE
+				else
+					return
```

Comment:
```suggestion
				S.destroy_everything = new_value == "CRUSH" ? TRUE : FALSE
```

Reply:
There's the if(new_value == null) case, where you just cancel out the input (no change). I could add in a quick if(isnull(new_value)) return above it if you'd like.